indexing-service/src/test/java/org/apache/druid/indexing/common/task/AppenderatorDriverRealtimeIndexTaskTest.java

```diff
@@ -324,9 +324,13 @@ public void testHandoffTimeout() throws Exception
     firehose.close();
 
     // handoff would timeout, resulting in exception
-    TaskStatus status = statusFuture.get();
-    Assert.assertTrue(status.getErrorMsg()
-                            .contains("java.util.concurrent.TimeoutException: Timeout waiting for task."));
+    try {
```

Comment:
this seems unrelated? is it a flaky test, perhaps? and this is a fix?

Reply:
I had a misunderstanding and updated the test now. Thanks for pointing it out.